### PR TITLE
Data description of QualityFilter

### PR DESCRIPTION
**Pull request by KennethEnevoldsen**
Data description of QualityFilter. 

This is a work in progress.

@mollerhoj I integrated the updated test, will you take a look at them?

### CODE REVIEW

**Review by mollerhoj — APPROVED** [review]
Looks fine. Though in the future I'd recommend not incorporating other PRs, it makes reviews a little harder IMO. What's the plan moving forward with this draft? What's missing? (Besides fixing the tests)?

File: src/examine_quality_filter.py/Repetitious_text.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
+"""
```
Comment (mollerhoj) [inline src/examine_quality_filter.py/Repetitious_text.py:1]: Any reason for the new `src` folder? :D

Comment (KennethEnevoldsen) [reply src/examine_quality_filter.py/Repetitious_text.py]: I actually went back and forth on this.

Basically, we kinda want two different code bases one which is simply the application, acting mostly as reference and documentation and the package which is intended to be reuseable.

Comment (KennethEnevoldsen) [reply src/examine_quality_filter.py/Repetitious_text.py]: I would love if you have an opinion on this, in other forks I have actually done it using src/dfm and src/application

**Pull request merged**